homework2/EditOrderStatusServlet.java

```diff
@@ -0,0 +1,75 @@
+package homework2;
+
+import lab3.FoodEntry;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+
+@WebServlet(loadOnStartup=1, urlPatterns={"/admin/order"})
+
+public class EditOrderStatusServlet extends HttpServlet {
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        int id = Integer.parseInt(request.getParameter("id"));
+        List<Order> entries = (List<Order>) getServletContext().getAttribute("orders");
+        Order leEntry = null;
+        for(Order entry: entries){
+            if(entry.getId() == id){
+                leEntry = entry;
+            }
+        }
+
+        response.setContentType("text/html");
+        PrintWriter out = response.getWriter();
+        out.println("<h1>Edit Order</h1>");
+        out.println("<form method =\"post\">");
+        out.println("Food Entry name: " + leEntry.getItem().getName() + "</br>");
+//        out.println("Price $: <input name='price' type='number' step='.01' value='"+ leEntry.getItem().getPrice() + "' /></br>");
+//        out.println("Customer: <textarea name='customerName'>" + leEntry.getCustomerName() + "</textarea></br>");
+//        out.println("<textarea name='description'>" + leEntry.getItem().getDescription() + "</textarea></br>");
```

Comment:
When editing, we also need to give admin ability to change name, price and so on.